### PR TITLE
Updated weddev.md to use new dart tool

### PR DESCRIPTION
**Pull request by Abhishek01039**

### CODE REVIEW

**Comment by Abhishek01039** [comment]
Hello @kwalrath 
can you please review this PR?
Thanks

File: src/tools/webdev.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The only time most web app developers run `build_runner` is for tests.
 
 <aside class="alert alert-info" markdown="1">
   **Dart 2 note:**
-  The `webdev` tool replaces the Dart 1.x `pub build` and `pub serve` commands.
+  The `webdev` tool replaces the Dart 1.x `dart pub build` and `dart pub serve` commands.
 </aside>
```
Comment (kwalrath) [inline src/tools/webdev.md:20]: Let's just delete this note. Dart 1.x is ancient history. :)

**Review by kwalrath — COMMENTED** [review]
Thanks, @Abhishek01039! LGTM after you delete the Dart 2 note.

(Did you test the commands? The changes seem straightforward, but I haven't worked on a web app in quite a while)

File: src/tools/webdev.md (hunk 1 above)
Comment (Abhishek01039) [reply src/tools/webdev.md]: Done

**Comment by Abhishek01039** [comment]
Yes, @kwalrath I have tried `dart pub global activate webdev` and it's work fine.

**Review by kwalrath — APPROVED** [review]

**Pull request merged**